FirebaseRemoteConfig/Sources/Public/FirebaseRemoteConfig/FIRRemoteConfig.h

```diff
@@ -325,12 +328,12 @@ NS_SWIFT_NAME(RemoteConfig)
 ///                         nil if the key doesn't exist in the default config.
 - (nullable FIRRemoteConfigValue *)defaultValueForKey:(nullable NSString *)key;
 
-#pragma mark - Realtime
+#pragma mark - Real-time Config Updates
 
 /// Completion handler invoked by `addOnConfigUpdateListener` when there is an update to
 /// the config from the backend.
 ///
-/// @param configUpdate Information on which key's values have changed
+/// @param configUpdate Information on which key's values have changed.
```

Comment:
Is it possible to link to `FIRRemoteConfigUpdate` here so folks have an easy link to the information? cc @morganchen12 is there `@link` syntax here?

Reply:
You can put a link directly into the header.

Reply:
Updated the comment. I think it should now have a link to the pragma mark for FIRRemoteConfigUpdate

Reply:
I don't see the link here?

Reply:
Instead of the link I'm using the double back-ticks to reference the `FIRRemoteConfigUpdate` method: https://stackoverflow.com/questions/38321880/how-to-go-about-adding-a-link-reference-to-another-method-in-documentation-xcode

Reply:
@morganchen12 can you confirm if the double back-ticks link to the method when the docs are generated?